LayoutTests/imported/w3c/web-platform-tests/css/css-writing-modes/forms/progress-appearance-native-computed-style.optional-expected.txt

```diff
@@ -1,6 +1,6 @@
 
 
 FAIL progress[style="writing-mode: horizontal-tb"] block size should match height and inline should match width assert_equals: expected "16px" but got "21px"
-PASS progress[style="writing-mode: vertical-lr"] block size should match width and inline should match width
-PASS progress[style="writing-mode: vertical-rl"] block size should match width and inline should match width
+FAIL progress[style="writing-mode: vertical-lr"] block size should match width and inline should match width assert_equals: expected "16px" but got "21px"
+FAIL progress[style="writing-mode: vertical-rl"] block size should match width and inline should match width assert_equals: expected "16px" but got "21px"
```

Comment:
Is there a reason we inconsistently apply the minimum block size?

Reply:
The test specifically has `assert_equals(style.blockSize, "16px");`, which seems unnecessary. We pass all of the important assertions:

```
assert_equals(style.blockSize, style.height);
assert_equals(style.inlineSize, style.width);
```

and 

```
assert_equals(style.blockSize, style.width);
assert_equals(style.inlineSize, style.height);
```

It was previously "passing" for `vertical-*`, since our native rendering code was not enforcing our default block size of 21px. We should probably remove the explicit size assertion from the test.

Reply:
I think the intent of that 16px assertion is to test the aspect ratio. Can we adjust the test so it tests the aspect ratio in a smarter way? (E.g. inline-size wider than block-size)

Reply:
Sounds good – I'll do that tomorrow in a separate PR.